src/index.css

```diff
@@ -148,3 +148,60 @@ button:focus-visible {
   margin-left: 4px;
   margin-right: 2px;
 }
+
+.article-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 80%;
+  max-width: 1500px;
+}
+
+.article-title {
+  font-size: 44px;
+  font-weight: 600;
+  line-height: normal;
+  margin-bottom: 0;
+}
+
+.article-info {
+  font-size: 13px;
+  display: flex;
+  color: #545658;
+  display: flex;
+  align-items: center;
+  margin-bottom: 0;
+}
+
+.article-topic {
+  padding: 8px 16px;
+  font-weight: 600;
+  background-color: #f6f6f6;
+  font-size: 18px;
+}
+
+.article-time {
+  margin-right: 4px;
+}
+
+.article-comment-icon {
+  margin-right: 4px;
+  margin-left: 4px;
+}
+
+.clock-icon {
+  margin-right: 4px;
+}
+
+.article-author {
+  font-weight: 600;
+  color: #141414;
+  font-size: 18px;
+  margin: 16px 0;
+}
+
+.article-body {
+  color: #141414;
```

Comment:
The css is still using a method of theming based on the users system theme. I have a dark theme which means the background colour is dark grey or black. Your article body is almost invisible on that colour. If you wish to stick to this paragraph colour you need a contrasting background colour on all devices.